src/Makefile.am

```diff
@@ -172,26 +172,27 @@ distclean-local:
 .libs/libpvecstatic.a: libpvecstatic.la
 .libs/libpvecdummy.a: libpvecdummy.la
 .libs/libpvecdummyPWR9.a: libpvecdummyPWR9.la
+.libs/libpvecdummyPWR10.a: libpvecdummyPWR10.la
 
 # libpvec definitions.
 # libpvec_la already includes vec_runtime_DYN.c compiled compiled -fpic
 # for IFUNC resolvers.
 # Now adding the -fpic -mcpu= target built runtimes.
 libpvec_la_LDFLAGS = -version-info $(PVECLIB_SO_VERSION)
 libpvec_la_LIBADD = vec_dynrt_PWR9.lo
-libpvec_la_LIBADD += vec_dynrt_PWR10.lo
 libpvec_la_LIBADD += vec_dynrt_PWR8.lo
 libpvec_la_LIBADD += vec_dynrt_PWR7.lo
+libpvec_la_LIBADD += vec_dynrt_PWR10.lo
 libpvec_la_LIBADD += vec_dynrt_common.lo
```

Comment:
well ... I realized that inserting at the top would always require 1 remove, and 2 adds. 

So I though it better to add PWR10 to end. But it would be a messy to reorder the list, So I did not.

But if this really bothers you ;) I will reorder the list (starting with PWR7 and ending with PWR10). Then we will be ready for PWR11.